frontend/tests/acceptance/pages/timeLogPage.js

```diff
@@ -1,4 +1,5 @@
 const { I } = inject();
+const { monthNames } = require('../helpers/calenderHelper');
```

Comment:
```suggestion
const { monthNames } = require('../helpers/calendarHelper');
```
You need to fix the spelling of calendar